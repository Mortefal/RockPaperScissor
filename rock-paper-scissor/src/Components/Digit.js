import React from 'react';

const DigitStyle = {
    fontSize: "48px"
}


class Digit extends React.Component {
  render() {
    return (
      <div>
        <label style={DigitStyle}>{this.props.digit}</label>
      </div>
    );
  }
}
export default Digit;
