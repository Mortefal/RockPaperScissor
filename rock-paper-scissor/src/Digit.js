import React from 'react';

class Digit extends React.Component{
    render() {
        return (
            <div>
                <label>{this.props.digit}</label>
            </div>
        );
    }
}
export default Digit;