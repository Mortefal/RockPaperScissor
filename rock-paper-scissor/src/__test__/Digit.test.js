import React from "react";
import renderer from 'react-test-renderer';
import Digit from "../Digit";

it('renders correctly', () => {
    const tree = renderer
        .create(<Digit/>)
        .toJSON();
    expect(tree).toMatchSnapshot();
});