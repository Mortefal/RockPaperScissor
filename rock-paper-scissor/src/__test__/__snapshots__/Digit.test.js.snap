// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <label
    style={
      Object {
        "fontSize": "48px",
      }
    }
  />
</div>
`;
